webhookbot/main.go

```diff
@@ -120,20 +120,14 @@ func main() {
 }
 
 func mainInner() int {
-	var opts Options
-	flag.StringVar(&opts.KeybaseLocation, "keybase", "keybase", "keybase command")
-	flag.StringVar(&opts.Home, "home", "", "Home directory")
-	flag.StringVar(&opts.Announcement, "announcement", os.Getenv("BOT_ANNOUNCEMENT"),
-		"Where to announce we are running")
-	flag.StringVar(&opts.HTTPPrefix, "http-prefix", os.Getenv("BOT_HTTP_PREFIX"),
+	opts := &Options{}
```

Comment:
Not a big fan of this, should have a `base.NewOptions()`

Reply:
yeah just adds a bunch of boilerplate unfortunately

Reply:
This case here just seems like it is incorrect anyway, shouldn't it be `opts := &base.Options{}`

Reply:
Oh wait nvm